common/src/test/resources/testConfig.json

```diff
@@ -20,7 +20,8 @@
           "aproxGroup": "aprox-group",
           "aproxGroupPublic": "aprox-group-public",
           "backupScmUrl": "backup-scm-url",
-          "cartographerServerUrl": "cartographer-server-url"
+          "cartographerServerUrl": "cartographer-server-url",
+          "defaultHttpRequestTimeout": 100000
```

Comment:
Because I was thinking about the situation, when we want to use set it also to other requests. If you want I can rename it